pages/coq-team.html

```diff
@@ -0,0 +1,503 @@
+    <div class="position">Associate Professor at IRIT, Toulouse University, France</div>
+    <div class="components">Primitive Types, Docker, Proof-General</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/palmskog.jpg" alt="Karl Palmskog" width="190px"/>
+    </div>
+    <div class="name"><a href="https://setoid.com/">Karl Palmskog</a></div><div class="nickname">palmskog</div>
+    <div class="position">Researcher at the KTH Royal Institute of Technology, Stockholm, Sweden</div>
+    <div class="role">Coq-community manager</div>
+    <div class="components">coq-community, opam, website, Roosterize</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/cpitclaudel.jpg" alt="Clément Pit-Claudel" width="170px"/>
+    </div>
+    <div class="name"><a href="http://pit-claudel.fr/clement/">Clément Pit-Claudel</a></div><div class="nickname">cpitclaudel</div>
+    <div class="position">PhD Student at MIT, MA, USA</div>
+    <div class="components">Documentation, Alectryon, Proof-General, Company-Coq</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/proux.jpg" alt="Pierre Roux" width="200px"/>
+    </div>
+    <div class="name"><a href="https://www.onera.fr/fr/staff/pierre-roux">Pierre Roux</a></div><div class="nickname">proux</div>
+    <div class="position">Researcher at Onera, Toulouse, France</div>
+    <div class="components">Primitive Types</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/pi8027.jpg" alt="Kazuhiko Sakaguchi" width="200px"/></div>
+    <div class="name">Kazuhiko Sakaguchi</div><div class="nickname">pi8027</div>
+    <div class="position">PhD Student at the University of Tsukuba, Japan</div>
+    <div class="components">Coercions, Extraction, Hierarchy Builder</div>
+  </div>   
+  <div class="person">
+    <div class="picture"><img src="/files/team/VincentSe.jpg" alt="Vincent Semeria" width="200px"/></div>
+    <div class="name">Vincent Semeria</div><div class="nickname">VincentSe</div>
+    <div class="position">Expert Software Developer at Finastra, Paris, France</div>
+    <div class="components">Reals</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/aspiwack.png" alt="Arnaud Spiwack" width="180px"/></div>
+    <div class="name"><a href="http://assert-false.science/arnaud/">Arnaud Spiwack</a></div><div class="nickname">aspiwack</div>
+    <div class="position">Senior Architect at Tweag, Paris, France</div>
+    <div class="components">Proof Engine, <verb>vm_compute</verb></div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/MSoegtropIMC.jpg" alt="Michael Soegtrop" width="200px"/></div>
+    <div class="name">Michael Soegtrop</div><div class="nickname">MSoegtropIMC</div>
+    <div class="position">Formal Verification Architect at Apple, Munich, Germany</div>
```

Comment:
Can you please remove my affiliation? I know it is public on LinkedIn but I still would prefer to not have it here.

Reply:
Sure.